frappe/public/js/frappe/form/controls/text_editor.js

```diff
@@ -232,7 +232,18 @@ frappe.ui.form.ControlTextEditor = frappe.ui.form.ControlCode.extend({
 		} catch(e) {
 			value = `<div class="ql-editor read-mode">${value}</div>`;
 		}
-
+		
+	    // quill keeps ol as a common container for both type of lists
+		// and uses css for appearances, this is not semantic
+		// so we convert ol to ul if it is unordered
+		value.find('ol li[data-list=bullet]:first-child').each((i, li) => {
+			let $li = $(li);
+			let $parent = $li.parent();
+			let $children = $parent.children();
+			let $ul = $('<ul>').append($children);
+			$parent.replaceWith($ul);
+		});
+		value = value.html();
```

Comment:
```suggestion
		let $value = $(value);
		$value.find('ol li[data-list=bullet]:first-child').each((i, li) => {
			let $li = $(li);
			let $parent = $li.parent();
			let $children = $parent.children();
			let $ul = $('<ul>').append($children);
			$parent.replaceWith($ul);
		});
		value = $value.html();
```

Reply:
I don't how it is working with your suggested code. It is not working for me.
Anyway, we can only call `.find()` on an element and the value is not an element we need to convert it to an element first.